src/doc/rustc/src/check-cfg/cargo-specifics.md

```diff
@@ -0,0 +1,96 @@
+# Cargo specifics - Checking conditional configurations
+
+<!--
+This page is the canonical place for describing the interaction between Cargo
+and --check-cfg, it is placed in the rustc book because --check-cfg and the
+unexpected_cfgs are owned by rustc, not Cargo. T-Cargo also considers
+`[lints.rust.unexpected_cfgs.check-cfg]` to be an implementation detail and
+is therefor not documented in Cargo.
+-->
+
+*See the Cargo announcement [blog post][check-cfg-blog-post] for more a global view of the feature.*
```

Comment:
blog posts are transient documentation, not evergreen.  Doesn't seem appropriate to link to it from docs

Reply:
My reasoning for including it, is that it has more details regarding the overall feature which are relevant for the next couple of releases (basicly I think until stable+1), after I agree that they will no longer be relevant and I would remove it.

Reply:
But I don't think it's that important, so I removed it.